doc/handling-high-traffic.md

```diff
@@ -0,0 +1,29 @@
+# Handling High Traffic
+
+## Introduction
+Starting in 2025 Wikibase cloud started occasionally receiving a large amount of traffic.
+
+This has caused a number of incidents:
+- https://wmde.github.io/wikibase-cloud-status/issues/2025-02-10-nginx-oom/ / T385969
+- https://wmde.github.io/wikibase-cloud-status/issues/2025-07-15-scrapers-again/ / T399439 and T400046
```

Comment:
Will GitHub automatically link the Phabricator tasks? If not, I think it would be nice to make them links.

Reply:
it does add nice links in commits but not in markdown